ludwig/schema/utils.py

```diff
@@ -390,6 +390,7 @@ def Boolean(default: bool, description: str, parameter_metadata: ParameterMetada
             "marshmallow_field": fields.Boolean(
                 truthy={True},
                 falsy={False},
+                validate=validate.OneOf([True, False]),
```

Comment:
Consider adding a comment about why this might be significant.

Reply:
added